src/pretty.ml

```diff
@@ -156,14 +156,14 @@ let pp_p_tactic : p_tactic pp = fun oc t ->
 let pp_command : p_command pp = fun oc cmd ->
   let out fmt = Format.fprintf oc fmt in
   match cmd.elt with
-  | P_require(m, P_require_default) ->
```

Comment:
You should output code as it was given by the used, so only one `require`. You can use `List.pp`. And the same comment holds for the other constructors (but see my comment about `require open` in `src/syntax.ml` first).